src/utils/fake-hash.ts

```diff
@@ -1,77 +1,77 @@
 import { range } from "lodash-es";
 import { randomByteArray } from "../api/zome-call-signing.js";
 import { DnaHash, ActionHash, AgentPubKey, EntryHash } from "../types.js";
-import { dhtLocationFrom32 } from "./hash-parts.js";
-
-async function fakeValidHash<T extends Uint8Array>(
```

Comment:
Since this function is not exported, it is safe to assume it was only used by this file.  Correct?